CHANGELOG.rst

```diff
@@ -7,6 +7,9 @@ Changelog
 .. note:: This version is not yet released and is under active development.
 
 * Added support for Python 3.6.
+* Changed ASN.1 dependency from pyasn1 to asn1crypto resulting in a general
```

Comment:
You can slap `pyasn1` and `asn1crypto` inside double-backticks so it doesn't try to spellcheck them.